addon/Feeds/LocalizationFeed.js

```diff
@@ -15,6 +16,10 @@ function getLocalizedStrings(locale, allStrings = STRINGS) {
   return Object.assign({}, allStrings[DEFAULT_LOCALE], strings || {});
 }
 
+function getDirection(locale) {
+  return (RTL_LIST.indexOf(locale.substr(0, 2)) >= 0) ? "rtl" : "ltr";
```

Comment:
We should split on "-" as some locales are actually 3 characters. (Although l10.js seems to compare the full locale string.)